pkg/resources/resource_source_kafka.go

```diff
@@ -45,7 +45,7 @@ var sourceKafkaSchema = map[string]*schema.Schema{
 		Optional:    true,
 		ForceNew:    true,
 	},
-	"include_headers": {
+	"include_header": {
```

Comment:
Same as the sasl mechanisms, in the docs it is `INCLUDE HEADERS`:

https://materialize.com/docs/sql/create-source/kafka/#headers